src/workerd/api/node/BUILD.bazel

```diff
@@ -115,7 +116,54 @@ wd_test(
 wd_test(
     src = "tests/crypto_keys-test.wd-test",
     args = ["--experimental"],
-    data = ["tests/crypto_keys-test.js"],
+    data = [
```

Comment:
We can simplify this and just use glob()

Reply:
It will be when done. Keeping it like this for now while making sure things are working.